vm/src/vm/macros.rs

```diff
@@ -102,9 +102,9 @@ macro_rules! decode3 {
     }};
 }
 
-macro_rules! compare_int {
-    ($vm:expr, $chunk:expr, $code:expr, $comp_fn:expr,
-     $compf_fn:expr, $wide:expr, $move:expr, $not:expr) => {{
+macro_rules! compare_numeric {
+    ($vm:expr, $chunk:expr, $code:expr, $comp_fn:expr, $wide:expr) => {{
+        let should_move = true;
```

Comment:
Why?  If $move was always true then get rid of this and collapse the if that uses it.  This was probably leftover from an earlier iteration.

Reply:
Fair point; I was hesitant to remove `$move` completely as I didn't understand what it was meant to convey. I'll clean it up and update the PR.